freud/diffraction.py

```diff
@@ -866,18 +882,22 @@ def plot(self, ax=None, cmap="afmhot", vmin=None, vmax=None):
         Returns:
             (:class:`matplotlib.axes.Axes`): Axis with the plot.
         """
-        if not _HAS_MPL:
-            raise ImportError(msg_mpl)
```

Comment:
To appease the pre-commit and the linter you want to revert the changes on the conditional imports to the code that is current with the main branch.